src/utils/UndoableActionsHandlerProvider.js

```diff
@@ -55,16 +54,13 @@ const BaseActionNotification = ({ action, removeAction, closeSnackbar, addSnackb
             key: action.id,
             options: {
                 persist: true,
-                // eslint-disable-next-line react/display-name
-                content: (key, message) =>
-                    <Notification
-                        message={message}
-                        actionText="Undo"
-                        actionHandler={handleCancel(key)}
-                        handleClose={handleClose(key)}
-                        onMouseEnter={handleMouseEnter}
-                        onMouseLeave={handleMouseLeave}
-                    />,
+            },
+            contentOptions: {
+                actionText: "Undo",
+                actionHandler: handleCancel(action.id),
+                handleClose: handleClose(action.id),
```

Comment:
Is passing `action.id` fine? This is not a raw prop anymore so it doesn't recognize `key`

Reply:
I'm not following, could you elaborate?

Reply:
We already spoke about this in the meeting but I had to specify `action.id` since this is not a raw prop anymore